locales/de.json

```diff
@@ -401,6 +401,7 @@
 				"student": "Schüler:in",
 				"studentParticipants": "Schüler / Teilnehmer",
 				"substituteTeacher": "Vertretungslehrer:in",
+				"noSubTeacherSelection": "Lehrer:in auswählen",
```

Comment:
avoid double revocation
```suggestion
				"TeacherSelection": "Lehrer:in auswählen",
```

Reply:
@luizzaly do we need these placeholders? If not I will remove them altogether